fastscanner/services/indicators/ports.py

```diff
@@ -42,6 +42,19 @@ def get(self) -> set[date]:
         ...
 
 
+class Channel(Protocol):
+    async def push(self, channel_id: str, data: dict[Any,Any], flush: bool = True):
+        ...
+
+    async def flush(self):
+        ...
+
+
+class PolygonRealtime(Protocol):
```

Comment:
PolygonRealtime is not a port. PolygonRealtime is an adapter. It's a specific implementation of the CandleStore port.